tests/integration_tests/cli_tests.py

```diff
@@ -335,111 +283,6 @@ def test_import_datasets_versioned_export(import_datasets_command, app_context,
     import_datasets_command.assert_called_with(expected_contents, overwrite=True)
 
 
-@mock.patch.dict(
-    "superset.config.DEFAULT_FEATURE_FLAGS", {"VERSIONED_EXPORT": False}, clear=True
```

Comment:
Try `superset.cli.lib.feature_flag` instead of `superset.config.DEFAULT_FEATURE_FLAGS`